README.md

```diff
@@ -4,6 +4,7 @@
 
 * [BilliardsTrainer](BilliardsTrainer.md)
 * [VidDiffusion](VidDiffusion.md)
+* [BudgetTravel](budgettravel.md)
```

Comment:
This list is supposed to be in alphabetic order.

Reply:
Fixed.